README.md

```diff
@@ -178,13 +178,16 @@ az pipelines create --name $PIPELINE_NAME_DOCKER --repository $REPOSITORY_NAME \
 
 ```shell
 PIPELINE_NAME_JMETER=jmeter-load-test
+ORGANIZATION_ADDRESS=https://dev.azure.com/YOUR_ORG_NAME
+PROJECT_NAME=your-project-name
 
 az pipelines create --name $PIPELINE_NAME_JMETER --repository $REPOSITORY_NAME \
     --repository-type tfsgit --branch master --skip-first-run \
     --yml-path pipelines/azure-pipelines.load-test.yml
 
-az pipelines variable create --pipeline-name $PIPELINE_NAME_JMETER --name TF_VAR_JMETER_JMX_FILE --allow-override
-az pipelines variable create --pipeline-name $PIPELINE_NAME_JMETER --name TF_VAR_JMETER_SLAVES_COUNT --allow-override
+az pipelines variable create --pipeline-name $PIPELINE_NAME_JMETER --name TF_VAR_JMETER_JMX_FILE --allow-override --organization $ORGANIZATION_ADDRESS --project $PROJECT_NAME

```

Comment:
@Guilhermeslucas 
I don't think we need the orgnization and project here, since we already set the defaults, as mentioned on the docs:

```shell
az devops configure --defaults organization=https://dev.azure.com/your-organization project=YourProject
```

Indeed, we could refactor that part and make the org/project more explicit in a dedicated variable.

Reply:
I agree @allantargino, another possible approach to simplify the experience could be add them on a bash script and explicitly mention on the documentation to update the two variables + run the script.

Reply:
I see! I am getting errors when running it without org and project. But I really like the approach you suggested. It fits the needs! ahhaha